src/model/NPMResolver.ts

```diff
@@ -42,20 +44,22 @@ export class NPMResolver {
         assetsPath: string
     ): boolean {
         if (fs.existsSync(NPMResolver.getPluginHashFilePath(assetsPath))) {
-            const packageLockUpdated = NPMResolver.packageLockWasUpdated(
+            const pluginPackageJsonPath =
+                NPMResolver.getPluginPackageJsonPath(assetsPath);
+            const pluginPackageJsonUpdated = NPMResolver.hashRootWasUpdated(
                 NPMResolver.getPluginHashFilePath(assetsPath),
-                NPMResolver.getPluginPackageLockPath(assetsPath),
+                pluginPackageJsonPath,
                 pluginName
             );
-            if (packageLockUpdated) {
+            if (pluginPackageJsonUpdated) {
                 console.log(
                     cgreen`⇢`,
-                    `[${pluginName}] (NPM) package-lock.json was updated...`
+                    `[${pluginName}] (NPM) package.json was updated...`
                 );
             }
             const hasNodeModules =
                 NPMResolver.getPluginNodeModulesPath(assetsPath);
-            if (!hasNodeModules || packageLockUpdated) {
+            if (!hasNodeModules || pluginPackageJsonUpdated) {
```

Comment:
I think the following case is not possible:
as a developer I work locally on cplace and a new FE2.0 artifact is available. On executing cplace-asc the latest artifact should be installed

- new cplace artifact available in jfrog -> no npm install is triggered by cplace-asc

node_modules -> `cf-platform@22.3.12` is installed

package.json:
```
"@cplace-next/cf-platform": "release-22.3",
```

package-lock.json:
```
{
"@cplace-next/cf-platform": {
  "version": "22.3.12",
  "resolved": "https://cplace.jfrog.io/artifactory/api/npm/cplace-npm/@cplace-next/cf-platform/-/@cplace-next/cf-platform-22.3.12.tgz",
  "integrity": "sha512-jaEs9rwdpNK5o80ej4poxLY0DU+NvUv7DkGcCYRvLeEvjTG9oB2JHrceuSn1mlyl1KXaQVWohZi2j7Of1Wpbxw=="
}
```

package.json.hash exists and is correct

-> 22.3.13 is not installed

Reply:
Probably there is two solutions:
- query jfrog for new artifacts 
- always execute `npm install`

Reply:
@stefancplace Good point! Let's discuss it in the daily

Reply:
is working now